modules/vaos/app/controllers/vaos/institutions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module VAOS
+  class InstitutionsController < VAOS::BaseController
```

Comment:
I'm not sure how I feel about "institution" as a description. This service is returning data about a clinic, which is at best an office within a facility, and sometimes just an abstract bucket for scheduling purposes. In other places "institutionCode" refers to a code or id for a facility, not a clinic, and similarly, in the data we get back in the array for this, the fields prefixed with institution refer to the facility the clinic is associated with.